cmd/discovery/main.go

```diff
@@ -95,7 +95,7 @@ func NewClient(config *types.DiscoveryConfig, stateTimeout time.Duration, logger
 	mode := ModeMaster
 	client := &cl.Client{}
 	var err error
-	if config.Slave {
+	if config.Slave { //If Starbuck -> Open GRPc Connection to Master (Apollo)
```

Comment:
The comment reads like it applies only to Starbuck, but it applies to all _followers_, doesn't it?